### PR TITLE
ComboBox: Fixing tab to lowerContent (accessibility issue)

### PR DESCRIPTION
**Pull request by kellygorr**
#### Pull request checklist

[0] Addresses an existing issue: Fixes #0000
[X] Include a change request file using `$ npm run change`

#### Description of changes
Allow user to set dismissMenu to True/False

Scenario
When lowerContent is added to the comboBox (onRenderLowerContent), navigating to the lowerContent is not possible, because the menu closes on TAB.  Calling dismissMenu(false) allows menu to stay open on TAB.
test1

Edit 11/16/17- Here are some more details about my scenario:
I am using the ComboBox, with a scrollable list of times (from 12 am to 12 pm), and a custom button which is below the scrolling content (added using onRenderLowerContent).  This would need to be tabbed to, so the user does not have to scroll through the long list, since the button is not within the scrollable options.

Edit 11/17/17- Added tests to show what I was able to get working (when tabbing to custom button)
Test 1: Prevents dismiss of menu and prevents focus returning to input (this was the only option that worked for me)
Test 2: No updates (this did not work)
Test 3:  Add focus to custom button on tab (this did not work)
test1

Edit 12/22/17 - Moved tests 2 and 3 to codepen.  Added test 1 (the only working one) to the end of the combobox examples.
https://codepen.io/kellygorr/pen/opzxLm

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx [hunk 1]
```diff
@@ -352,9 +352,9 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    * Close menu callout if it is open
    */
   @autobind
-  public dismissMenu(): void {
+  public dismissMenu(dismissMenu = true): void {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx]: I think you want the default to be false. It is a more predictable behavior. Also you need some better description of what the flag is intended to do. It doesn't really make sense to call `dismissMenu(false)` and expect it to stay open...

**Comment by dzearing** [comment]
Hmm. Why is the menu dismissing on tab? It should only dismiss on losing focus somewhere, but you should be able to tab around within the menu. Maybe this is the wrong fix.

**Comment by betrue-final-final** [comment]
Tab? You arrow inside a menu, not tab.

**Comment by kellygorr** [comment]
Hi @betrue-final-final !  Here are some more details about my scenario (I will add this above too):
I am using the ComboBox, with a scrollable list of times (the Callout), and a custom button which is below the scrolling content (added using onRenderLowerContent).  This would need to be tabbed to, so the user does not have to scroll through the long list, since the button is not within the scrollable options.

**Comment by betrue-final-final** [comment]
Oh!, then it does seem like @dzearing is right.

**Comment by kellygorr** [comment]
@dzearing @betrue-final-final 
Hi!  I added some tests to the top of the examples page that show what I did to make the custom button tabable.  It shows what I got to work  (Test 1), and what didn't work.  As-is, tabing to the custom button didn't work (Test 2).  Setting focus to the custom button on-tab didn't work, the menu closes and focus goes to the next input (Test 3).

**Comment by joschect** [comment]
@kellygorr The combo box should control the focus and there should be a way for the combobox to "set" focus to the custom content and then get it back. The pickers do this a little bit.

**Comment by dzearing** [comment]
@kellygorr have you sync'd with @jspurlin here? Can we close this PR until you have the issues resolved, or do you plan to make updates? We want to make sure things aren't sitting around unattended, so we can have a better scope on what PRs need attention.

**Comment by kellygorr** [comment]
@dzearing  We've been having discussions via email, but they have kind of fizzled.  I sent emails on 2/14 and 2/21 to try and get details from June Lee about the accessibility direction for this (@jspurlin is also on these emails), but she hasn't responded.  The keyboarding recommendation from my team is to tab to the custom button, so that is what I would go with.  I will close this PR for now, it would probably be easier to start from scratch anyway.  In a future update I'd go with Jeremy's recommendation to "update the aria-activedecendant" when tabbing to the custom button.

**Pull request closed without merging**